pkg/proxy/parser.go

```diff
@@ -312,6 +310,7 @@ func (p *Parser) parseInputState(b []byte) []byte {
 		case "n":
 			p.confirmStatus.SetStatus(StatusNone)
 			p.srvOutputChan <- []byte("\r\n")
+			p.TerminalParser.resetCommand()
 			p.command = ""
 			return p.breakInputPacket()
 		default:
```

Comment:
I'm sorry, but there seems to be no code you're requesting modifications on. Could please specify which part from the provided code do you want me to review?